jp/components/spreadsheet_configuring.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Angular Spreadsheet | Grid Controls | Ignite UI for Angular | Configuration | Infragistics |
+_description: Use the Spreadsheet control to embed Excel document creation and editing experiences right into your application.
+_keywords: Spreadsheet, Ignite UI for Angular, Infragistics
+---
+
+## Configuring Spreadsheet
+
+The `IgxSpreadsheet` allows the user to configure many different aspects of the control, including but not limited to editing of the cells, the visibility of gridlines and headers, protection, zoom level, and various other properties related to the Excel worksheet. 
+
+### Demo
+
+<div class="sample-container" style="height: 500px">
+    <iframe id="spreadsheet-overview-sample-iframe" src='{environment:demosBaseUrl}/spreadsheet/spreadsheet-configuring' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn"   data-iframe-id="spreadsheet-overview-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">View on StackBlitz
```

Comment:
Since all Configs for Spreadsheet samples are commented: [line](https://github.com/IgniteUI/igniteui-angular-samples/pull/1084/files#diff-d649ae0325ca5c21fad1f639fa8dce11R110), should we provide Stackblitz buttons for these samples?

Reply:
currently there are issues with stackblitz and the spreadsheet / excel